primo/utils/setup_logger.py

```diff
@@ -21,22 +22,37 @@
 
 # User defined libs
 from primo.utils.raise_exception import raise_exception
-from primo.utils import LOGGER_FORMAT, LOGGER_DATE
 
+LOGGER_FORMAT = "%(asctime)s - %(name)s:%(lineno)d - %(levelname)s - %(message)s"
+LOGGER_DATE = "%d-%b-%y %H:%M:%S"
 
-def setup_logger(log_file: pathlib.Path, log_level: int, log_to_console: bool):
+
+class LogLevel(enum.Enum):
+    """Enum for logging levels"""
+
+    CRITICAL = logging.CRITICAL
+    WARNING = logging.WARNING
```

Comment:
Agree. I wanted to avoid the multiple if-else statements, so created this one. I now reverted it to the old format. In place of multiple if-else statements, I add a dictionary to perform the check.

Reply:
:-). You could have used the case statement if multiple if-else were bothering you. Either way, the above works for me!